recipes/rdkit/bld.bat

```diff
@@ -0,0 +1,51 @@
+rem surely there's a better way than this
+if "%PY_VER%"=="2.7" (
+	set PYTHON_LIBRARY=python27.lib
```

Comment:
Well if we know `python` will be included, we can certainly parse this with `python`. Maybe something like this (untested)?

```batch
python -c "import sys; print('python%i%i.lib' % sys.version_info[:2])" > temp.txt
set /p PYTHON_LIBRARY=<temp.txt
```

We do something similar in [the `boost` feedstock]( https://github.com/conda-forge/boost-feedstock/blob/b9c7173ec7908ba9f8dd01fd5916ad68e8acd5c8/recipe/bld.bat#L27-L29 ).

Reply:
I'll probably leave this for only only since I know it works, but we could clean that up later. That comment was left over from the earlier rdkit channel recipes :)